### DISCUSSION TITLE
Summarize your question into a single sentence here

### DISCUSSION THREAD

**Discussion by eliyash31** [GriefPrevention/GriefPrevention#969] [Q&A] [answered]
Hello, how can I view or change the numbers into player names? 

or is this really how the plugin runs its mysql database? like it wont really show the player names? if not then please help how can I reset the database and make it show the player's names instead of these numbers or id.

I really need to know the player name because our server resets every 6 months along with the claim blocks but we have certain ranks or perks that claims or total claim blocks of the players remain.. thanks for those who can help :) 

here is the url for the sample image: https://imgur.com/mhRIL56

**Comment by Jikoo** [comment, marked as answer]
Data is stored by UUID, just like everything else in Minecraft should be. Your permissions are also probably stored by UUID, you just don't see it because you're not poking through the internals.

Are you manually resetting everything? That sounds like a huge waste of your time. You should probably consider writing a plugin to handle that for you, which would also vastly simplify the process of using a UUID to identify a player.
